Installation/include/CGAL/config.h

```diff
@@ -346,7 +346,10 @@ using std::max;
 #define CGAL_NORETURN  [[noreturn]]
 
 // Macro to specify [[no_unique_address]] if supported
-#if __has_cpp_attribute(no_unique_address)
+#if _MSC_VER >= 1929
+// see https://devblogs.microsoft.com/cppblog/msvc-cpp20-and-the-std-cpp20-switch/#c20-no_unique_address
+#  define CGAL_NO_UNIQUE_ADDRESS [[msvc::no_unique_address]]
```

Comment:
In this [testsuite](https://cgal.geometryfactory.com/CGAL/testsuite/CGAL-5.6-Ic-225/AABB_tree/TestReport_Christo_MSVC-2022-Community-Release.gz) we get the warning
```
include\CGAL/Lazy.h(1170,3): warning C4848: support for attribute 'msvc::no_unique_address' in C++17 and earlier is a vendor extension
``` 
Is there a fix needed, or shall we suppress the warning?

Reply:
We could use:
```patch
diff --git a/Installation/include/CGAL/config.h b/Installation/include/CGAL/config.h
index c88c0c5a32c..b4b65e75e18 100644
--- a/Installation/include/CGAL/config.h
+++ b/Installation/include/CGAL/config.h
@@ -346,7 +346,7 @@ using std::max;
 #define CGAL_NORETURN  [[noreturn]]
 
 // Macro to specify [[no_unique_address]] if supported
-#if _MSC_VER >= 1929
+#if _MSC_VER >= 1929 && _MSVC_LANG >= 202002L
 // see https://devblogs.microsoft.com/cppblog/msvc-cpp20-and-the-std-cpp20-switch/#c20-no_unique_address
 #  define CGAL_NO_UNIQUE_ADDRESS [[msvc::no_unique_address]]
 #elif __has_cpp_attribute(no_unique_address)
```